docs/reference/esql/esql-async-query-api.asciidoc

```diff
@@ -1,27 +1,19 @@
 [[esql-async-query-api]]
-== {esql} async query API
+=== {esql} async query API
 ++++
 <titleabbrev>{esql} async query API</titleabbrev>
 ++++
 
-Runs an async {esql} search.
+Runs an async <<esql,{esql} query>>.
 
-The async query API lets you asynchronously execute a search request,
-monitor its progress, and retrieve results as they become available.
+The async query API lets you asynchronously execute a query request,
+monitor its progress, and retrieve results when they become available.
 
-Executing an <<esql,ES|QL ({es} query language)>> is commonly quite fast,
-however searches across large data sets or frozen data can take some time.
-To avoid long waits, run an async {esql} search.
+Queries initiated by this API may return results or not.
```

Comment:
Maybe "Just like the sync API, queries initiated..." -- not sure what is meant with this sentence, but can also stay :).

Reply:
I was trying to get across the point that results OR an id would be returned. This point is better clarified later in the spec. I'll just remove this sentence from here as it's a little confusing.